do not fetch rss automatically, respect some privacy!

use update_checks_enabled option to tune the default (altho new option would be
more reasonable)

--- nagios/html/main.php~	2012-05-29 00:04:23.000000000 +0300
+++ nagios/html/main.php	2012-05-29 00:08:36.419474072 +0300
@@ -25,6 +25,10 @@ $this_year = '2018';
 		var vboxText = "<a href=https://www.nagios.com/tours target=_blank> " +
 						"Click here to watch the entire Nagios Core 4 Tour!</a>";
 	<?php } ?>
+<?php
+	$updateinfo=get_update_information();
+	if ($updateinfo['update_checks_enabled']!==false):
+?>
 	$(document).ready(function() {
 		var user = "<?php echo $_SERVER['REMOTE_USER']; ?>";
 
@@ -31,7 +34,9 @@ $this_year = '2014';
 			'#splashbox5-contents', 'corepromo', 3,
 			'<ul>', processPromoItem, '</ul>'
 		);
-
+<?php
+		endif;
+?>
 		getCoreStatus();
 	});
 
@@ -141,7 +141,7 @@ $this_year = '2014';
 
 <div id="updateversioninfo">
 <?php
-	$updateinfo = get_update_information();
+	//$updateinfo = get_update_information();
 	if (!$updateinfo['update_checks_enabled']) {
 ?>
 		<div class="updatechecksdisabled">
